build/.gitmodules

```diff
@@ -0,0 +1,56 @@
+[submodule "third_party/zlib"]
```

Comment:
be careful on this. Please don't include this addition of submodules in your final change.